gateway/src/apicast/policy/standalone/configuration.lua

```diff
@@ -31,6 +31,10 @@ do
     local tostring = tostring
     local path = require('pl.path')
     local file = require('pl.file')
+    local inspect = require('inspect')
+    local tab_clone = require "table.clone"
+    local __tostring = function(self) return inspect(tab_clone(self)) end
```

Comment:
Why do we need the clone?

Reply:
We need to get rid of the metatable, otherwise, it does infinite recursion. Will mention it in a comment.